spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-sentinel/src/main/java/com/alibaba/cloud/sentinel/feign/SentinelFeign.java

```diff
@@ -77,20 +79,25 @@ public Feign build() {
 				@Override
 				public InvocationHandler create(Target target,
 						Map<Method, MethodHandler> dispatch) {
-					// using reflect get fallback and fallbackFactory properties from
-					// FeignClientFactoryBean because FeignClientFactoryBean is a package
-					// level class, we can not use it in our package
-					Object feignClientFactoryBean = Builder.this.applicationContext
-							.getBean("&" + target.type().getName());
-
-					Class fallback = (Class) getFieldValue(feignClientFactoryBean,
-							"fallback");
-					Class fallbackFactory = (Class) getFieldValue(feignClientFactoryBean,
-							"fallbackFactory");
-					String beanName = (String) getFieldValue(feignClientFactoryBean,
-							"contextId");
+
+					GenericApplicationContext gctx = (GenericApplicationContext) Builder.this.applicationContext;
+					BeanDefinition def = gctx.getBeanDefinition(target.type().getName());
+
+					/**
+					 * TODO
+					 * 由于初始化顺序发生变更，这里为了避免循环依赖，只能通过 BeanDefinition 的方式获得 FeignClientFactoryBean
+					 * 需要重点review
+					 */
+//					FeignClientFactoryBean feignClientFactoryBean = (FeignClientFactoryBean) Builder.this.applicationContext
```

Comment:
Change to Chinese comments

Reply:
Sorry, my mistake, it has been changed to English